src/pages/sponsor-and-support.js

```diff
@@ -0,0 +1,109 @@
+            <p>Sponsorships made to Gastown Gang will grow all the events under its umbrella.</p>
+
+            <h2>Types of sponsorship</h2>
+            <p>You or your organization can be either a <em>venue</em> or a <em>cash</em> sponsor.</p>
+
+            <h3>Venue</h3>
+            <p>Venue sponsors may chose to host one or several of Gastown Gang-supported events.</p>
+
+            <h4>What you get:</h4>
+            <ul>
+              <li>A personalized mention by our emcee at each event.</li>
+              <li>Two minutes to speak at the start of the event. One of your employees or developer relations person should speak — no recruiters! You may prepare something for us to read. You may include job opportunities during your two minutes, but please don’t recruit audience members during socialization time.</li>
+              <li>A space at the event to give out knickknacks and info.</li>
+            </ul>
+            <ul>Your logo and our thanks in three emails:
+              <li>To all past attendees 1 week before the event you're sponsoring.</li>
+              <li>Reminding RSVPd attendees about the event you're sponsoring.</li>
+              <li>Thanking the attendees for attending the event you sponsored.</li>
+            </ul>
+
+            <h3>Cash sponsors</h3>
+            <p>Gastown Gang offers one benefits tier to any and all cash sponsors. Sponsorship funds will be used for food, drinks, and operational costs of the partner meetups and Gastown Gang initiatives. You can sponsor either 1 month at full rate or multiple months at increasingly discounted rates.</p>
+
+            <table>
+              <thead>
+                <tr>
+                  <th>Packages</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>1 month</td>
+                  <td>3 months</td>
+                  <td>6 months</td>
+                  <td>12 months</td>
+                </tr>
+                <tr>
+                  <td>$500</td>
+                  <td>$1350</td>
+                  <td>$2550</td>
+                  <td>$4800</td>
+                </tr>
+                <tr>
+                  <td>(full rate)</td>
+                  <td>save 10% </td>
+                  <td>save 15%</td>
+                  <td>save 20%</td>
+                </tr>
+              </tbody>
+            </table>
+
+            <h4>What you get for the duration of your support:</h4>
+            <p><strong>All benefits given to Venue sponsors</strong><em> plus</em></p>
+            <ul>
+              <li>Invitations for your employees to attend.</li>
+              <li>If you're a recurring sponsor, we can share your knickknacks and info materials if your employees cannot attend an event.</li>
+              <li>Your logo on our website <strong>and</strong> the websites of all our partner events.</li>
+              <li>Your logo and our thanks on our Twitter once per week.</li>
```

Comment:
I wonder if weekly sponsor shout-outs might be a bit much...

Thoughts?

Reply:
The perks were inspired by donutJS. It could be helpful to ask them for feedback about their experience giving them 🤔 I'm reluctant to have things slowed down by waiting on external factors, but still could be worth it.

Reply:
Weekly shoutouts removed!